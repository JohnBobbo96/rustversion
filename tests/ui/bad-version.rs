#[rustversion::since(stable)]
struct S;

fn main() {}
