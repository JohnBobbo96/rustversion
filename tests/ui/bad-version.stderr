error: expected rustc release number, like 1.31
 --> $DIR/bad-version.rs:1:22
  |
1 | #[rustversion::since(stable)]
  |                      ^^^^^^
